### ISSUE TITLE
Feature request: capturing available hours

### ISSUE THREAD

**Issue by inger** [backlogs/redmine_backlogs#811]
platform: 2.2.2 # supported: 2.0.4, 2.1.6, 2.2.2
backlogs: 0.9.35 # supported: 0.9.35
ruby: 1.9.3 # supported: 1.9.3, 1.8.7

As a team, I'd like to capture hours available in beginning of the planning, so that we can better allocate stories and tasks for the sprint. 

Sometimes we do not fill the sprint up to %100, and during the sprint we'd like to see the burndown against the available hours.

As part of the planning, we first come up with the available hours in a sprint (affected by holidays, etc..), and typically this number is higher than the sum of all the work as we don't fill up the sprint 100%. 

I'd like to record these hours in RBL, for example on a sprint properties popup, start the burndown ideal line at this level, while the remaining hours graph would still start according to the work estimates.

Simply put, I'd offset to the ideal line - ideally by (available-remaining) hours. If available not given default back to remaining. Discussion with Friflaj: http://forum.redminebacklogs.net/capturing-available-hours-td4025335.html#a4025392
